content/java/concepts/array-list/terms/contains/contains.md

```diff
@@ -0,0 +1,57 @@
+---
+Title: '.contains()'
+Description: 'To check if an element is present in the list or not.'
+Subjects:
+  - 'Code Foundations'
+  - 'Computer Science'
+Tags:
+  - 'Arrays'
+  - 'Data Types'
+CatalogContent:
+  - 'learn-java'
+  - 'paths/computer-science'
+---
+
+The `.contains()` method is used for checking if the specified element is present in `ArrayList` or not. It will return true if the element is present; otherwise, it will return false.
+
+## Syntax
+
+To check if an element is present in `ArrayList`, it is passed as an argument in the `.contains()` method of `ArrayList`. The return value is of the boolean type. True if the element is present; otherwise, false if it is not present. More formally, it returns true if and only if this `ArrayList` contains at least one element `e` such that `(o==null? e==null: o.equals(e))`. Here `o` is the object passed to the list, and `e` is an element of the list.
+
+```pseudo
+arrayListInstance.contains(o);
```

Comment:
```suggestion
arrayListInstance.contains(obj);
```

Reply:
I wrote it `o` because I mention it in the paragraph.
`(o==null? e==null: o.equals(e))` 
I think it will be better to change `o` to `obj` in the paragraph as well.